toolkit/tools/pkg/imagecustomizerlib/imagecustomizer.go

```diff
@@ -223,6 +228,29 @@ func hasPartitionCustomizations(config *imagecustomizerapi.Config) bool {
 	return config.Disks != nil
 }
 
+func validateAdditionalFiles(baseConfigPath string, additionalFiles *imagecustomizerapi.AdditionalFilesMap) error {
+	var aggregateErr error
+	for sourceFile := range *additionalFiles {
+		sourceFileFullPath := filepath.Join(baseConfigPath, sourceFile)
+		isFile, err := file.IsFile(sourceFileFullPath)
+		if err != nil {
+			aggregateErr = imagecustomizerapi.AggregateErrors(aggregateErr, fmt.Errorf("invalid AdditionalFiles source file (%s):\n%w", sourceFile, err))
+		}
+
+		if !isFile {
+			aggregateErr = imagecustomizerapi.AggregateErrors(aggregateErr, fmt.Errorf("invalid AdditionalFiles source file (%s): not a file", sourceFile))
+		}
+	}
+	return aggregateErr
+}
+
+func validateIsoConfig(baseConfigPath string, config *imagecustomizerapi.Iso) error {
+	if config == nil {
+		return nil
+	}
+	return validateAdditionalFiles(baseConfigPath, &config.AdditionalFiles)
```

Comment:
nit: Don't omit the `if err != nil` for the final operation in a function. For example, if other fields are added to `imagecustomizerapi.Iso`, then it'll make updating this function more difficult.

Reply:
fixed.